cli/test/smokehouse/config/config.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
just to make file search easier for the core config stuff, maybe call this something like `exclusions.js`? I also think it's fine in the parent directory along side e.g. `core-tests.js`, but up to you